packages/manager/apps/web/client/app/app.js

```diff
@@ -30,7 +30,9 @@ import ovhManagerCatalogPrice from '@ovh-ux/manager-catalog-price';
 import ovhManagerNavbar from '@ovh-ux/manager-navbar';
 import ovhManagerProductOffers from '@ovh-ux/manager-product-offers';
 import uiRouter, { RejectType } from '@uirouter/angularjs';
-import ovhManagerServerSidebar from '@ovh-ux/manager-server-sidebar';
+import ovhManagerServerSidebar, {
+  ORDER_URLS,
+} from '@ovh-ux/manager-server-sidebar';
```

Comment:
Thinking: I don't know what is the best design : put all order urls in the same constants files and share it between all packages or have dedicated url in each module and expose them.

Reply:
@cbourgois I think the current issue is that modules are not all clearly defined (for example it's not possible to import stuff from hosting module without having to install web manager)